frontend/packages/console-app/src/components/quick-starts/QuickStartDrawer.tsx

```diff
@@ -60,25 +62,35 @@ const QuickStartDrawer: React.FC<QuickStartDrawerProps> = ({
     return onClose();
   };
 
+  const handleScroll = (e: React.UIEvent<HTMLDivElement>) =>
+    setHeaderStyles(
+      classNames(
+        'co-quick-start-drawer-head',
+        e.currentTarget.scrollTop && 'pf-u-box-shadow-md-bottom',
+      ),
+    );
+
   const panelContent = quickStart ? (
-    <DrawerPanelContent>
-      <DrawerHead>
-        <div className="co-quick-start-drawer__title">
-          <Title
-            headingLevel="h1"
-            size="xl"
-            style={{ marginRight: 'var(--pf-global--spacer--md)' }}
-          >
-            {quickStart?.spec.displayName}
-          </Title>
-          <Title headingLevel="h6" size="md" className="text-secondary">
-            {`${quickStart?.spec.duration} minutes`}
-          </Title>
-        </div>
-        <DrawerActions>
-          <DrawerCloseButton onClick={handleClose} />
-        </DrawerActions>
-      </DrawerHead>
+    <DrawerPanelContent onScroll={handleScroll}>
```

Comment:
If we are going with this approach. I would suggest using the hook `useScrollDirection` from [console-shared](https://github.com/openshift/console/blob/master/frontend/packages/console-shared/src/hooks/scroll.ts). Take a look at it

Reply:
Didn't know about `useScrollDirection`. Updated.

Reply:
TBH, I don't really like this approach of using scroll position to add some styles but unfortunately there's no clean CSS solution for this as well. I guess it's okay to do it like this but I wish there was some CSS pseudo selector for sticky elements. 

FYI there is a CSS solution to this. You can add a shadow to the sticky element and add a separate cover element that covers this shadow in normal position and shows the shadow when scrolled up.

Reply:
> TBH, I don't really like this approach of using scroll position to add some styles but unfortunately there's no clean CSS solution for this as well. I guess it's okay to do it like this but I wish there was some CSS pseudo selector for sticky elements.

I agree here that a CSS pseudo selector solution would be good. I looked around on web and found the same solution that you mentioned with css. But that's not a clean solution and I think current solution is okay. So, I guess we can keep the current solution.